spec/features/course/unread_status_management_spec.rb

```diff
@@ -39,7 +40,8 @@
     end
 
     describe 'sidebar' do
-      let!(:announcement) { create(:course_announcement, course: course, creator: second_user) }
+      let!(:announcement) { create(:course_announcement, course: course, creator: second_user,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.